### PR TITLE
chore: add NOTES.txt for fullstack-deployment helm chart

### LINKED ISSUE #321: Charts[Docs] Create Notes.txt for hedera-network helm chart
### Requested Feature

Need to update NOTES.txt in `charts/hedera-network` 

### Business Need

NOTES.txt in helm chart includes useful details that an end user relies on. It should include endpoints and ports that a user can expose to access.

For example, if user deploys the telemetry-stack locally, they will need to port-forward the Grafana and Prometheus services so that those are accessible locally.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by qnswirlds**
## Description

This pull request changes the following:

- Add NOTES.txt for fullstack-deployment helm chart

### Related Issues

- Closes #321

### CODE REVIEW

File: charts/fullstack-deployment/templates/NOTES.txt [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+
+
+
+
+Special note:
+In the future, the fullstack network manager CLI tool will also support this port-forwarding, as follows:
+  npm install fullstack-network-manager
+  fsnetman expose hedera-explorer ${LOCAL_PORT}
+  fsnetman expose grpc -node 0 ${LOCAL_PORT}
```
Comment (qnswirlds) [inline charts/fullstack-deployment/templates/NOTES.txt:33]: @leninmehedy, I think these commands are better than those in https://github.com/hashgraph/full-stack-testing/issues/420#issuecomment-1788318504. Please revise. Thanks!

Comment (leninmehedy) [reply charts/fullstack-deployment/templates/NOTES.txt:33]: Let's remove the Special Note section for now until those are finalized. You would need to make another PR later once we have the CLI ready

**Review by leninmehedy — APPROVED** [review]
Please remove the Special Note section for now and merge.

File: charts/fullstack-deployment/templates/NOTES.txt (hunk 1 above)
Comment (qnswirlds) [reply charts/fullstack-deployment/templates/NOTES.txt]: Done

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**